templates/bot-msgext/js/default/README.md

```diff
@@ -1,76 +1,71 @@
 
-A _Messaging Extension_ allows users to interact with your web service through buttons and forms in the Microsoft Teams client. They can search, or initiate actions, in an external system from the compose message area, the command box, or directly from a message.
+Messaging Extensions are implemented on top of the Bot support architecture within Teams.
 
-This is a simple hello world application with both Bot and Message extension capabilities. You can learn how to build a Teams app with Bot and Message Extension at the same time.
+A bot, chatbot, or conversational bot is an app that responds to simple commands sent in chat and replies in meaningful ways. Examples of bots in everyday use include: bots that notify about build failures, bots that provide information about the weather or bus schedules, or provide travel information. A bot interaction can be a quick question and answer, or it can be a complex conversation. Being a cloud application, a bot can provide valuable and secure access to cloud services and corporate resources.
+
+This is a simple hello world application with both Bot and Message extension capabilities.
 
 ## Prerequisites
 
-- [Node.js](https://nodejs.org/en/)
-- An M365 account, if you do not have M365 account, apply one from [M365 developer program](https://developer.microsoft.com/en-us/microsoft-365/dev-program)
-- Teams Toolkit or TeamsFx CLI
+- [NodeJS](https://nodejs.org/en/)
+- An M365 account. If you do not have M365 account, apply one from [M365 developer program](https://developer.microsoft.com/en-us/microsoft-365/dev-program)
+- [Teams Toolkit Visual Studio Code Extension](https://aka.ms/teams-toolkit) or [TeamsFx CLI](https://aka.ms/teamsfx-cli)
 
 ## Create an application
 
-- From Visual Studio Code, there are two ways to create a new bot and messaging extension, select `Create New Project` in the left panel or directly open the command palette and select `Teams: Create New Project`.
-- From TeamsFx CLI, run command `teamsfx new` to create a new bot and messaging extension.
+- From Visual Studio Code, open command palette and select `Teams - Start a new project`. Or from the CLI, (after `npm install -g teamsfx`) run command `teamsfx new`.
+- Choose the messaging extension capability from the prompts.
   > Note: You have the option to reuse an existing bot by entering the credential manually. But make sure that bot is not associated with any AAD apps.
 
 ## Debug
 
-Start debugging the project by hitting the `F5` key. Alternatively use the `Run Panel` in Visual Studio Code and click the `Start Debugging` green arrow button.
+Start debugging the project by hitting the `F5` key in Visual Studio Code. Alternatively use the `Run and Debug Activity Panel` in Visual Studio Code and click the `Start Debugging` green arrow button.
```

Comment:
I prefer to remove "in Visual Studio Code", for F5 is on keyboard, not in vscode.